src/app/main/component/ubs/components/ubs-confirm-page/ubs-confirm-page.component.html

```diff
@@ -1,31 +1,46 @@
 <div class="wraper">
-    <div class="order">
-        <h2 class="title">Дякуємо за замовлення</h2>
-        <div class="description">
-            <p>
-                Ми вдячні Вам за бережливе ставлення до навколишнього середовища.
-                Якщо ви зацікавлені в інших послугах або корисній інформації, 
-                завітайте за нашими корисними посиланнями.
-            </p>
-        </div>
-        <div class="row infolinks">
-            <div class="col-md-6 list">
-                <ul>
-                    <h5>УБС сервіси</h5>
-                    <li><a>Зробити інше замовлення</a></li>
-                    <li><a>Пластик Поштою</a></li>
-                    <li><a>Сайт Україна Без Сміття</a></li>
-                    <li><a>Еко магазин</a></li>
-                </ul>
-                <ul>
-                    <h5>Корисна інформація</h5>
-                    <li><a>Інструкція з переробки сміття</a></li>
-                    <li><a>Відео про сортування і переробку</a></li>
-                </ul>
-            </div>
-            <div class="col-md-6">
-                <img class="image" src="assets/img/noNews.jpg" alt="eco-people">
-            </div>
-        </div>
+  <div class="order">
+    <h2 class="title">{{ 'confirm-page.title-text' | translate }} (№{{ orderId }})</h2>
+    <div class="description">
+      <p>{{ 'confirm-page.description' | translate }}</p>
     </div>
+    <div class="row infolinks">
+      <div class="col-md-6 list">
+        <ul>
+          <h5>{{ 'confirm-page.ubsServices' | translate }}</h5>
+          <li>
+            <a>{{ 'confirm-page.other-order' | translate }}</a>
+          </li>
+          <li>
+            <a>{{ 'confirm-page.mail' | translate }}</a>
+          </li>
+          <li>
+            <a>{{ 'confirm-page.ukraine-site' | translate }}</a>
+          </li>
+          <li>
+            <a>{{ 'confirm-page.eco-shop' | translate }}</a>
+          </li>
+        </ul>
+        <ul>
+          <h5>{{ 'confirm-page.information' | translate }}</h5>
+          <li>
+            <a>{{ 'confirm-page.instruction' | translate }}</a>
+          </li>
+          <li>
+            <a>{{ 'confirm-page.video' | translate }}</a>
+          </li>
+        </ul>
+      </div>
+      <div class="col-md-6">
+        <img class="image" src="assets/img/noNews.jpg" alt="eco-people" />
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12">
+        <p class="p-40">
+          {{ 'confirm-page.questions' | translate }} <a class="link" href="">{{ 'confirm-page.contact' | translate }}</a>
+        </p>
+      </div>
+    </div>
+  </div>
```

Comment:
looks like responsive breacks on 2500px screens
![Screenshot 2021-06-08 at 20 02 24](https://user-images.githubusercontent.com/46226026/121227717-8fe81a00-c894-11eb-925f-957d30cbb8f9.png)

Reply:
also, provide PR description
and PR title should contain ticket number